data/json/furniture_and_terrain/furniture-migo.json

```diff
@@ -83,7 +83,7 @@
     "type": "furniture",
     "id": "f_alien_scar",
     "name": "scarred lump",
-    "description": "This is a pile of unidentified twitching alien flesh, belching strange gases out of injured vessels.",
+    "description": "This is a pile of nondescript alien flesh, twitching and belching strange gases out of injured orofices.",
```

Comment:
```suggestion
    "description": "This is a pile of nondescript alien flesh, twitching and belching strange gases out of injured orifices.",
```

Reply:
Whoops, nice catch!

Reply:
Looks like this one accidentally didn't get merged.